Documentation/artifact-format.md

```diff
@@ -8,6 +8,82 @@ Note that there are some restrictions on ordering of the files, described
 in the "Ordering" section.
 
 
+### version 3
+
+```
+-artifact.mender (tar format)
+  |
+  +---version
+  |
+  +---manifest
+  |
+  +---manifest.sig
+  |
+  +---manifest-augment
```

Comment:
Now that I think of it, what do you think of renaming `manifest-augment` to `manifest-unsigned`? It might make it more obvious what it is for. And same for the header.

Reply:
Disagree. You don't need to have a signature at all. Yet still you will use some "extra" parameters in those augment files.